### DISCUSSION TITLE
İbax yeni dünya 🔥💸

### DISCUSSION THREAD

**Discussion by Byjokers** [IBAX-io/go-ibax#829] [Soru Cevap - Türkçe]
IBAX ağını kullanarak DEFI veya DEX'i hızlı bir şekilde kurabilirsiniz ve yalnızca IBAX akıllı sözleşmeleri yazmanız ve ön ucu tasarlamanız yeterlidir.

**Comment by Yusufknk** [comment]
Geleceği birlikte inşa edeceğiz.(İBAX kuruluş felsefesi)
